scraperwiki.php

```diff
@@ -0,0 +1,159 @@
+<?php
```

Comment:
It would be much better if this and the other library files weren't included in the repository and was installed by composer like other scrapers. You can see how to make this change here: https://morph.io/documentation/scraperwiki